Common/src/main/java/com/blamejared/crafttweaker/api/ScriptLoadingOptions.java

```diff
@@ -1,19 +1,28 @@
 package com.blamejared.crafttweaker.api;
 
+import com.blamejared.crafttweaker.api.zencode.bracket.IgnorePrefixCasingBracketParser;
+import com.blamejared.crafttweaker.api.zencode.impl.CraftTweakerDefaultScriptRunConfiguration;
+import com.blamejared.crafttweaker.platform.Services;
 import net.minecraft.resources.ResourceLocation;
+import org.openzen.zencode.java.ScriptingEngine;
+import org.openzen.zencode.java.module.JavaNativeModule;
+import org.openzen.zencode.shared.CompileException;
+import org.openzen.zenscript.parser.BracketExpressionParser;
 
 import javax.annotation.Nonnull;
+import java.util.function.BiConsumer;
+import java.util.function.Consumer;
 
 public class ScriptLoadingOptions {
     
     public static final ScriptLoadSource RELOAD_LISTENER_SCRIPT_SOURCE = new ScriptLoadSource(CraftTweakerConstants.rl("reload_listener"));
     public static final ScriptLoadSource CLIENT_RECIPES_UPDATED_SCRIPT_SOURCE = new ScriptLoadSource(CraftTweakerConstants.rl("client_recipes_updated"));
     
-    
     private boolean format;
     private boolean execute;
     private String loaderName = CraftTweakerConstants.DEFAULT_LOADER_NAME;
     private ScriptLoadSource source = RELOAD_LISTENER_SCRIPT_SOURCE;
+    private LegacyRunConfig runConfiguration = this::legacyConfig;
```

Comment:
We may need to think about the namings:

We **load Scripts** 
via **Script Runs** 
that have **ScriptLoadingOptions**
which rely on a **LoaderName** and a **RunConfig**